CONTRIBUTING.md

```diff
@@ -26,133 +26,169 @@ We specifically do not merge PRs **without prior issues** that:
 - Fix linter warnings for tools not currently in the CI pipeline
 - Add new CI tooling
 
-## Long version
+---
+
+## Writing code for OpenTofu
 
-This repository contains OpenTofu Core, which includes the command line interface, the main graph engine, and the documentation for them.
+Eager to get started on coding? Here's the short version:
 
-This document provides guidance on OpenTofu contribution recommended practices. It covers how to submit issues, how to get involved in the discussion, how to work on the code, and how to contribute code changes.
+1. Set up a Go development environment with Git.
+2. Pay attention to copyright: write the code yourself, avoid copy/paste. Disable or limit your AI coding assistant.
```

Comment:
it may be worth linking to the DCO here for more information too.

Reply:
I agree